### ISSUE TITLE
Slack enabled but seems not working

### ISSUE THREAD

**Issue by minhdanh** [alerta/alerta-contrib#31]
I added slack to the plugin list (by adjusting `PLUGINS` value) and added SLACK_WEBHOOK_URL environment variable. From the server log I could see that Slack is already loaded:

```
2016-05-26 15:20:06,836 - alerta.app[22523]: INFO - Server plug-in 'normalise' enabled.
2016-05-26 15:20:06,839 - alerta.app[22523]: INFO - Server plug-in 'reject' enabled.
2016-05-26 15:20:06,842 - alerta.app[22523]: INFO - Server plug-in 'slack' enabled.
2016-05-26 15:20:06,845 - alerta.app[22523]: INFO - Server plug-in 'normalise' enabled.
2016-05-26 15:20:06,847 - alerta.app[22523]: INFO - Server plug-in 'reject' enabled.
2016-05-26 15:20:06,850 - alerta.app[22523]: INFO - Server plug-in 'slack' enabled.
```
But after adding alerts to alerta, I never see any message posted to the desired slack channel.

**Comment by minhdanh** [comment]
It turns out that slack plugin did override the channel with `#alerts` (which doesn't exist in my case).
The default slack webhook url is attached to a channel so it's not necessary for a hard-coded channel. It would be nice if this channel is configurable via an environment variable.

**Comment by minhdanh** [comment]
The url to the alert (http://try.alerta.io/#/alert) should also be configurable, too.

**Comment by satterly** [comment]
Thanks for the feedback. I'll make the changes you suggest.

**Comment by minhdanh** [comment]
@satterly I submitted a pull request : D
https://github.com/alerta/alerta-contrib/pull/33

**Comment by satterly** [comment]
Fixed by #33
